runconfig/opts/opts.go

```diff
@@ -46,6 +46,17 @@ func doesEnvExist(name string) bool {
 	return false
 }
 
+// ValidateArg validates a build-arg variable and returns it.
+// Build-arg is in the form of <varname>=<value> where <varname> is required.
+func ValidateArg(val string) (string, error) {
+	arr := strings.Split(val, "=")
+	if len(arr) > 1 && len(arr[0]) > 0 {
```

Comment:
The condition is a little cryptic to read – it hard to read the *intent* of it. Might be good to have a small method that just checks for an empty string `if len(arr) > 1 && isNotEmpty(arr[0])`.